lib/util/bit_array.c

```diff
@@ -21,11 +20,6 @@ typedef uint64_t spdk_bit_array_word;
 #define SPDK_BIT_ARRAY_WORD_INDEX_SHIFT	spdk_u32log2(SPDK_BIT_ARRAY_WORD_BITS)
 #define SPDK_BIT_ARRAY_WORD_INDEX_MASK	((1u << SPDK_BIT_ARRAY_WORD_INDEX_SHIFT) - 1)
 
-struct spdk_bit_array {
```

Comment:
Let's keep the definiton here, it appears to have been kept private by purpose.

Reply:
Done